_data/security.yml

```diff
@@ -117,6 +117,15 @@ websites:
           text: "Only available for enterprise accounts."
       doc: https://engineering.opendns.com/2014/05/22/launching-two-step-verification/
 
+    - name: Pwnie Express Pwn Pulse
+      url: https://www.pwnieexpress.com/products/pulse-device-detection
+      img: pwnieexpress.png
+      tfa: Yes
+      software: Yes
+      exceptions:
+          text: "TFA Can be enabled on Pwn Pulse Accounts upon customer request."
+      doc: https://www.pwnieexpress.com/support
```

Comment:
@jamcat22's direct link looks like the best option, but being a PDF is not ideal (for mobile users). Perhaps adding a note saying "Under "Pulse" click "User Guide" and scroll to page 41." would be a suitable alternative.